app/stores/sighting.js

```diff
@@ -94,5 +94,43 @@ module.exports = {
         this.get({'source': source}, function (status, response) {
             callback(status, response);
         });
+    },
+
+    /*
+     * get the pokemon sightings within a specific time range
+     */
+    getByTimeRange : function (req, callback) {
+        let range = (req.range) || '1d',
+            fromTs = new Date(req.ts),
+            toTs;
+
+        let rangeValue = parseInt(range, 10),
+            rangeSpan = range.replace(/\d+/g, '');
+
+        switch(rangeSpan) {
+            // week
+            case 'w':
+                toTs = new Date(fromTs.getTime() + rangeValue * 7 * 24 * 60 * 60 * 1000);
```

Comment:
These values are not configurable and these are used for time calculation, so I think it's not required to put into config file.